src/h5web/vis-packs/core/utils.ts

```diff
@@ -120,16 +120,21 @@ export function getDomain(
 export function extendDomain(
   bareDomain: Domain,
   extendFactor: number,
-  isLog?: boolean
+  scaleType: ScaleType = ScaleType.Linear
```

Comment:
I think the scale could be mandatory and come before the `extendFactor`, since we now pass variables as the first two parameters and a constant as the third.

Reply:
Not sure about this, I think it makes sense to have a "light" version of `extendDomain` that do not need any `scaleType` for simpler uses.

Also, that would imply having a fallback for `abscissaScale` in the `LineVis`.

Reply:
ah right, `abscissaScale` is optional, got you 👍